### DISCUSSION TITLE
请问PicGo最新版是不是不支持32位的win7

### DISCUSSION THREAD

**Discussion by heracleshebe147** [Molunerfinn/PicGo#686] [Q&A]
如题所示，下载了PicGo-Setup-2.3.0-beta.6.exe，安装时提示要64位才行，请问PicGo最新版是不是不支持32位的win7？

还是说有别的版本，或者说再等等

**Comment by Molunerfinn** [comment]
不支持32位windows。未来会考虑增加32位版本的构建。不过还是建议升级至64位系统使用。
